pages/product.js

```diff
@@ -1,53 +1,58 @@
+const PriceHelper = require('../helpers/priceHelper');
+const priceHelper = new PriceHelper();
+const ElementCheckingHelper = require('../helpers/elementCheckingHelper');
+const elCheck = new ElementCheckingHelper();
 const { I } = inject();
-
-
 module.exports = {
     colorDropDown: { xpath: '//label[text()="Color"]/following-sibling::div/a[1]' },
     sizeDropDown: { xpath: '//label[text()="Size"]/following-sibling::div/a[1]' },
     colorOption: { xpath: '//label[text()="Color"]/following-sibling::div/ul/li[2]/a' },
     sizeOption: { xpath: '//label[text()="Size"]/following-sibling::div/ul/li[2]/a' },
     productPriceText: { xpath: '//*[@id="content"]/div[1]/div[2]/div/div[1]/span' },
+    available: { xpath: '//*[@id="content"]/div[1]/div[2]/div/ul/li[last()]' },
 
-     parsePrice(priceString) {
-        return parseFloat(priceString.replace(/[^0-9.-]/g, ''));
+    async checkAvailable() {
```

Comment:
rename to `verifyAvailable`, `check` should return true/false, not to throw errors